public/examples/js/offscreen-canvas/basic.js

```diff
@@ -0,0 +1,45 @@
+// This example is the based on demos-basic/container, but using OffscreenCanvas.
+
+const width = 800;
+const height = 600;
+const resolution = window.devicePixelRatio;
+const canvas = document.createElement('canvas');
+canvas.style.width = `${width}px`;
+canvas.style.height = `${height}px`;
+document.body.appendChild(canvas);
+const view = canvas.transferControlToOffscreen();
+
+const app = new PIXI.Application({
+    width, height, resolution, view, backgroundColor: 0x1099bb,
```

Comment:
I really want to try to simplify that examples as much as possible. Can you remove width, height, resolution. And use `background: '#1099bb'` instead?

Reply:
Fixed!